source/_front.md

```diff
@@ -4,6 +4,9 @@ This boils down to three operating regimes: single-microsecond latency *within a
 
 ![Overview](/assets/latency-domains.png)
 
-[**Join SymbioticLab**](https://forms.gle/L3Syau9dBzi8eLxQ7) to work on new and exciting projects with us!
+<!-- [**Join SymbioticLab**](https://forms.gle/L3Syau9dBzi8eLxQ7) to work on new and exciting projects with us! -->
+[**Learn about ongoing projects**](https://docs.google.com/document/d/1mVPqfnqLz_CXVz8XyDLcB1BIlUyezY2A-t5mV33JZHw/edit?usp=sharing) Are you seeking collaboration and already in UMich? Learn about our exciting ongoing projects.
```

Comment:
There should be a colon after the Learn about ongoing projects. Join us didn't have one because it was a running sentence. 

The actual sentence is also weird. I'd just say, "If you're already a Michigan undergrad or Master's student, you can learn more about our ongoing projects and opportunities here" (here has the link instead of linking on the header)